tasks/create_ordered_kafka_groups.yml

```diff
@@ -1,56 +1,10 @@
 ---
-# Run these tasks over kafka_broker group with serial: 1
```

Comment:
seems like this task file should not even exist anymore... wherever its getting invoked we can just call the role

Reply:
Yes, it was only used from the `upgrade_kafka_broker.yml` playbook. But in order to provide a non-breaking change, I left it there - in case someone was using the task-file.

Reply:
hmm good point. we dont really support running those tasks files, so you are safe to remove it. I'd rather not have a dead file lying around

Reply:
Ok, I'll delete it then. Then we can remove the role variable defaults for the group names. WDYT?

Reply:
@domenicbove I deleted the "dead file" and removed the introduced role variables - as they are no longer needed. PTAL